src/frontend/src/optimizer/plan_node/generic/source.rs

```diff
@@ -117,9 +117,16 @@ impl Source {
             sub_fields: vec![],
             type_name: "".to_string(),
         };
+        let update_time = Field {
```

Comment:
Will this affect the schema of the already created source/table? Is this change backward-compatible?

Reply:
If not, I suggest we add the update time field in the json.

Reply:
This pr won't merge, yesterday we found that debezium can capture the event time on upstream, so we decide to generate a (RW process time - event time) metric to promethues as an indication of lagging. FYI
https://github.com/risingwavelabs/risingwave/issues/13440